projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/interestrate/future/provider/InterestRateFutureOptionMarginSecurityNormalSmileMethod.java

```diff
@@ -21,7 +21,8 @@
  * The normal parameters are represented by (expiration-strike-delay) surfaces. The "delay" is the time between option expiration and future last trading date,
  * i.e. 0 for quarterly options and x for x-year mid-curve options. The future prices are computed without convexity adjustments.
```

Comment:
Review the Javadoc. The normal parameters are now (expiration, delay, strike, futures price) dependent.